examples/plot_mealy.py

```diff
@@ -58,18 +58,18 @@
 print("R2 = %.2f" % r2_score)
 
 ##############################################################################
-# Fit a Model Performance Predictor on the model performances.
+# Fit an Error Tree on the model performances.
 
 error_analyzer = ErrorAnalyzer(model, feature_names=feature_names)
 error_analyzer.fit(X_test, y_test)
 
 ##############################################################################
-# Print metrics regarding the Model Performance Predictor.
+# Print metrics regarding the Error Tree.
 
 print(error_analyzer.evaluate(X_test, y_test, output_format='text'))
 
 ##############################################################################
-# Plot the Model Performance Predictor Decision Tree.
+# Plot the Error Decision Tree.
```

Comment:
[Extreme nitpick]
Can be simply `Error Tree` as we use this term everywhere else? Not very important, for consistency